### DISCUSSION TITLE
get script

### DISCUSSION THREAD

**Discussion by herokarimpoor** [Breakthrough/PySceneDetect#225] [General]
I need a script that recognizes scenes and gets frames from scenes.
I can use these git codes?

Thanl so much

**Comment by Breakthrough** [comment]
Not out of the box, but you could build such a script using PySceneDetect + OpenCV or PyAV.

[See `api_test.py`](https://github.com/Breakthrough/PySceneDetect/blob/master/tests/api_test.py) for an example of how to get the start/end frames on a video.  Then, you can loop through a video and save the frames yourself.

Hope this helps!

**Comment by herokarimpoor** [comment]
Thanks for your good guidance
You can also run a script to identify the scene
